apps/src/dance/Dance.js

```diff
@@ -37,6 +37,9 @@ import firehoseClient from '@cdo/apps/lib/util/firehose';
 import {showArrowButtons} from '@cdo/apps/templates/arrowDisplayRedux';
 import danceCode from '@code-dot-org/dance-party/src/p5.dance.interpreted.js';
 import utils from './utils';
+import ErrorBoundary from '@cdo/apps/lab2/ErrorBoundary';
+import Lab2MetricsReporter from '@cdo/apps/lab2/Lab2MetricsReporter';
```

Comment:
This is a basic question - but why are we using `Lab2MetricsReporter` here instead of `MetricsReporter`? @sanchitmalhotra126

Reply:
Discussed offline - `Lab2MetricsReporter` adds a common field `appName`. `MetricsReporter` is intended to be feature agnostic so lab/project-specific fields weren't added there.